JustAnotherVoiceChat.Server.Wrapper/src/Elements/Client/VoiceClient.cs

```diff
@@ -66,6 +68,16 @@ public bool SetNickname(string nickname)
             return Server.NativeWrapper.SetClientNickname(this, nickname);
         }
 
+        private T RunWhileConnected<T>(Func<T> callback)
```

Comment:
I don't know if `RunWhileConnected` is the right name for this method. It implies that it runs as long as the player is connected.